<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
    <addaction name="actionNewProject"/>
    <addaction name="actionOpenProject"/>
    <addaction name="actionSaveProject"/>
   </widget>
   <widget class="QMenu" name="menuWork_W">
    <property name="title">
     <string>Setting(&amp;S)</string>
    </property>
    <addaction name="actionCalibrationBoard"/>
    <addaction name="actionImages"/>
    <addaction name="actionCamera"/>
    <addaction name="actionParameters"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionTheme"/>
   </widget>
   <widget class="QMenu" name="menuTool_T">
    <property name="title">
     <string>Tool(&amp;T)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView_V">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuDocks_D">
     <property name="title">
      <string>Docks(&amp;D)</string>
     </property>
     <addaction name="actionViewImage"/>
     <addaction name="actionViewOutput"/>
     <addaction name="actionViewRepErrImage"/>
     <addaction name="actionViewRepErrPoint"/>
     <addaction name="actionViewCamera"/>
    </widget>
    <addaction name="menuDocks_D"/>
   </widget>
   <widget class="QMenu" name="menuWindow_W">
    <property name="title">
     <string>Window(&amp;W)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWork_W"/>
   <addaction name="menuTool_T"/>
   <addaction name="menuView_V"/>
   <addaction name="menuWindow_W"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProject"/>
   <addaction name="actionOpenProject"/>
   <addaction name="actionSaveProject"/>
   <addaction name="separator"/>
   <addaction name="actionCalibrationBoard"/>
   <addaction name="actionImages"/>
   <addaction name="actionCamera"/>
   <addaction name="actionParameters"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="separator"/>
   <addaction name="actionTheme"/>
   <addaction name="separator"/>
   <addaction name="actionHelp"/>
   <addaction name="actionAbout"/>
  </widget>
  <action name="actionNewProject">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/new.png</normaloff>:/pic/new.png</iconset>
   </property>
   <property name="text">
    <string>New Project(&amp;N)</string>
   </property>
  </action>
  <action name="actionOpenProject">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/open.png</normaloff>:/pic/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Project(&amp;O)</string>
   </property>
  </action>
  <action name="actionSaveProject">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/save.png</normaloff>:/pic/save.png</iconset>
   </property>
   <property name="text">
    <string>Save Project(&amp;S)</string>
   </property>
  </action>
  <action name="actionCalibrationBoard">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/CameraCalibration.png</normaloff>:/pic/CameraCalibration.png</iconset>
   </property>
   <property name="text">
    <string>CalibrationBoard(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>Set Calibration Board Parameters</string>
   </property>
  </action>
  <action name="actionImages">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/Pictures.png</normaloff>:/pic/Pictures.png</iconset>
   </property>
   <property name="text">
    <string>Images(&amp;I)</string>
   </property>
   <property name="toolTip">
    <string>Add images from folder for calibration</string>
   </property>
  </action>
  <action name="actionCamera">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/Camera.png</normaloff>:/pic/Camera.png</iconset>
   </property>
   <property name="text">
    <string>Camera(&amp;C)</string>
   </property>
   <property name="toolTip">
    <string>Add images direct from camera for calibration</string>
   </property>
  </action>
  <action name="actionParameters">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/parameters.png</normaloff>:/pic/parameters.png</iconset>
   </property>
   <property name="text">
    <string>Parameters(&amp;P)</string>
   </property>
   <property name="toolTip">
    <string>Set parameters for calibration</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/Start.png</normaloff>:/pic/Start.png</iconset>
   </property>
   <property name="text">
    <string>Start(&amp;S)</string>
   </property>
   <property name="toolTip">
    <string>Start Calibration</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/Stop.png</normaloff>:/pic/Stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop(&amp;T)</string>
   </property>
   <property name="toolTip">
    <string>Stop Calibration</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/Help.png</normaloff>:/pic/Help.png</iconset>
   </property>
   <property name="text">
    <string>Help(&amp;H)</string>
   </property>
   <property name="toolTip">
    <string>Help for how to use</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/about.png</normaloff>:/pic/about.png</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="toolTip">
    <string>About this program</string>
   </property>
  </action>
  <action name="actionViewImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Image(&amp;I)</string>
   </property>
  </action>
  <action name="actionViewOutput">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Output(&amp;O)</string>
   </property>
  </action>
  <action name="actionViewRepErrImage">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RepErrImage(&amp;G)</string>
   </property>
  </action>
  <action name="actionViewRepErrPoint">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RepErrPoint(&amp;P)</string>
   </property>
  </action>
  <action name="actionViewCamera">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Camera(&amp;C)</string>
   </property>
  </action>
  <action name="actionTheme">
   <property name="icon">
    <iconset resource="pic.qrc">
     <normaloff>:/pic/color_wheel.png</normaloff>:/pic/color_wheel.png</iconset>
   </property>
   <property name="text">
    <string>Theme(&amp;M)</string>
   </property>
   <property name="toolTip">
    <string>Theme for display</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="pic.qrc"/>
 </resources>
 <connections/>
</ui>
